proposals/0053-quic-turbine.md

```diff
@@ -0,0 +1,120 @@
+---
+simd: '0053'
+title: Block Propagation Using QUIC Protocol
+authors:
+  - Solana Labs
+category: Standard
+type: Networking
+status: Draft
+created: 2023-05-31
+feature: (fill in with feature tracking issues once accepted)
+---
+
+## Summary
+
+Use QUIC protocol (instead of current UDP implementation) to propagate blocks
+across cluster.
+
+## Motivation
+
+* The Solana protocol currently uses an datagram-based block distribution
+  mechanism (Turbine/UDP) using out-of-band peer discovery.
+* We find that Turbine/UDP lacks important features to support reliable block
+  data distribution over WAN, namely:
+  * Turbine/UDP does not specify congestion control mechanism and thus lacks
+    the ability to dynamically adapt to packet corruption/loss (currently
+    mitigated via forward error correction with a hardcoded ratio).
+  * Turbine/UDP lacks data authentication and a mechanism to reject incoming
+    block data streams.
+  * This makes Turbine/UDP recipients susceptible to packet floods.
+  * Commercial DDoS protection solutions are considered ineffective for custom
+    UDP protocols like Turbine/UDP.
+* QUIC is an Internet standard defining a protocol for a UDP-based multiplexed,
+  authenticated, and encrypted transport.
+  * QUIC is already in use in other parts of the Solana protocol.
+  * QUIC implements many of the features required that Turbine/UDP currently
+    lacks.
+  * We expect wider specialization of network infrastructure towards QUIC from
```

Comment:
For effective management of QUIC protocol this would require border gateways with ability to decrypt the traffic, which means that without a separate network identity the protocol/node identity would need to be used.